slamon_afm/tests/afm_test.py

```diff
@@ -27,3 +27,31 @@ def setUp(self):
     def tearDown(self):
         db.drop_all()
         self.app_context.pop()
+
+
+class ConfigUtilTest(TestCase):
+    def testOkTrueValues(self):
+        for value in ('True', 'TRUE', 'true', 'Yes', 'yes', 'YES'):
+            self.assertTrue(_bool_from_str(value))
+
+    def testOkFalseValues(self):
+        for value in ('False', 'FALSE', 'false', 'No', 'no', 'NO'):
+            self.assertFalse(_bool_from_str(value))
+
+    def testNokValues(self):
+        for value in ('f', 't', '0', '1', 'nope', 'yep'):
+            with self.assertRaises(ValueError):
+                _bool_from_str(value)
+
+    def testOkLogLevels(self):
+        # known values (standard logging levels)
+        for value in ('CRITICAL', 'Error', 'warning', 'INFO', 'DEBUG', 'NOTSET'):
+            self.assertEqual(getattr(logging, value.upper()), _log_level_from_str(value))
+        # explicit integer values
+        for value in (-1, 0, 1, 50,100):
```

Comment:
How about testing with stringified integers: `("-1", "0", "1", "50", "100")`? You are trying to test `return int(value)` and it should always work with values that are integer to begin with... 

Maybe add another for-loop for strings if you want to keep this case here.